<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPropsDialog</class>
 <widget class="QDialog" name="SPropsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Section information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="simscanner.qrc">
    <normaloff>:/icons/icons/To_do_list.png</normaloff>:/icons/icons/To_do_list.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSectionProperties">
      <attribute name="title">
       <string>Section properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tblSection">
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Section Name</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Characteristics</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Virtual Address</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Memory size</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Physical Offset</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Raw size</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Section Entropy</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Comment</string>
          </property>
          <property name="background">
           <color>
            <red>192</red>
            <green>192</green>
            <blue>192</blue>
           </color>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lvSectionDump"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSectionEntropy">
      <attribute name="title">
       <string>Section Entropy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gboxEntropy">
         <property name="title">
          <string>Entropy</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frameEntropyMode">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="rbEntropy">
               <property name="text">
                <string>Entropy view</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbDerivative">
               <property name="text">
                <string>Derivative view</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frameEntropy">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxExtremumsDensity">
         <property name="title">
          <string>Extremums Density</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frameExtremumsDensityMode">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="rbMaximums">
               <property name="text">
                <string>Maximums view</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbMinimums">
               <property name="text">
                <string>Minimums view</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbExtremums">
               <property name="text">
                <string>All of extremums</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frameDensity">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionView_Section_associated_neuroprofiles">
   <property name="icon">
    <iconset resource="simscanner.qrc">
     <normaloff>:/icons/icons/nn.png</normaloff>:/icons/icons/nn.png</iconset>
   </property>
   <property name="text">
    <string>View Section associated neuroprofiles</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="simscanner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SPropsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SPropsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
